easybuild/easyconfigs/p/PANDAseq/PANDAseq-2.11-intel-2018a.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'ConfigureMake'
+
+name = 'PANDAseq'
+version = '2.11'
+
+homepage = 'https://github.com/neufeld/pandaseq'
+description = """PANDASEQ is a program to align Illumina reads, optionally with PCR primers embedded in the sequence, and reconstruct an overlapping sequence."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+# download from https://github.com/neufeld/pandaseq/archive/v2.11.tar.gz
```

Comment:
@ajaust Please remove this comment (gets outdated easily when this easyconfig is used for updates)